os/arch/arm/src/amebasmart/Kconfig

```diff
@@ -81,7 +81,19 @@ endif
 config AMEBASMART_I2C
 	bool "AMEBASMART_I2C"
 	default n
+
 if AMEBASMART_I2C
+
+config RTL8730E_I2C0
+	bool "I2C_0"
+	default n
+config RTL8730E_I2C1
+	bool "I2C_1"
+	default n
+config RTL8730E_I2C2
```

Comment:
minor comment.

These are RTL8730E, but others are AMEBASMART. When do you use `RTL8730E`?

Reply:
This happened since in TP1x, most of the components used AMEBASMART, only UART is using RTL8xxx.
I will correct this PR to use AMEBASMART too, UART will remain the same.